extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/core/RedisClient.java

```diff
@@ -0,0 +1,81 @@
+package org.prebid.server.hooks.modules.com.confiant.adquality.core;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.redis.client.RedisAPI;
+import io.vertx.redis.client.Response;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.BidScanResult;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.OperationResult;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.RedisBidsData;
+
+import java.util.Collections;
+import java.util.List;
+
+public class RedisClient {
```

Comment:
Also, why do you hardcode redis implementation, looks like this is not RedisClient, this is BidsScanner.